src/Netrc.php

```diff
@@ -21,6 +21,25 @@
  */
 class Netrc
 {
+    /**
+     * Get the default path of the netrc file that will be used if one
+     * is not provided.
+     *
+     * @return string
+     */
+    public static function getDefaultPath() {
+        $homePath = getenv('HOME');
+        if (!$homePath) {
+            return false;
```

Comment:
Since this method should return only strings (not booleans), i guess we should not return false here - wdyt if we throw FileNotFoundException here instead and add @throw line to doc block?

Reply:
I think I did that later, so sounds good to me.  Let me get you an updated patch.